src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+
+    @Override
+    public List<Machine> get() {
+        List<Machine> bulldozers = new ArrayList<>();
```

Comment:
BulldozerProducer should return the list of only Bulldozers. If we have Machine list we could put there other machine children and BulldozerProducer will return list which contains not only bulldozers. And this will be wrong work of BulldozerProducer .

Reply:
I agree. Done